modules/geo/geom.js

```diff
@@ -1,17 +1,4 @@
-
-import { geomLineIntersection} from '@id-sdk/geom';
-
-import {
-    vecAngle, vecDot, vecEqual,
-    vecLength, vecSubtract
-} from '@id-sdk/math';
-
-
-// Return the counterclockwise angle in the range (-pi, pi)
-// between the positive X axis and the line intersecting a and b.
-export function geoAngle(a, b, projection) {
```

Comment:
Unsure right at the moment why this method is getting toasted- was it not used anyplace? Or its use obviated by some combo of methods in `geom`?